src/sentry/static/sentry/app/components/forms/textField.tsx

```diff
@@ -1,16 +1,10 @@
-import PropTypes from 'prop-types';
-
 import InputField from 'app/components/forms/inputField';
 
 type Props = InputField['props'] & {
   spellCheck?: string;
 };
 
 export default class TextField extends InputField<Props> {
-  static propTypes = {
-    spellCheck: PropTypes.string,
-  } as any; // TODO(ts): remove when proptypes are no longer required, some views don't implement all required proptypes of underlying InputField.
```

Comment:
We're at the point where we should probably start removing propTypes, this makes typing easier since it won't try and typecheck the proptypes.